superset/db_engine_specs/base.py

```diff
@@ -1303,6 +1303,28 @@ def get_column_spec(
             )
         return None
 
+    @classmethod
+    def get_cancel_query_payload(cls, cursor: Any, query: Query) -> Any:
+        """
+        Returns None if query can not be cancelled.
+        :param cursor: Cursor instance in which the query will be executed
+        :param query: Query instance
+        :return: Type of the payload can vary depends on databases
+        but must be jsonable. None if query can't be cancelled.
+        """
+        return None
+
+    @classmethod
+    def cancel_query(cls, cursor: Any, query: Query, payload: Any) -> None:
+        """
+        Cancels query in the underlying database.
+        The method is called only when payload is not None.
+        :param cursor: New cursor instance to the db of the query
+        :param query: Query instance
+        :param payload: Value returned by get_cancel_query_payload or set in
+        other life-cycle methods of the query
+        """
```

Comment:
Doesn't this need `pass`? CI seems to be fine with it so maybe there's some python trivia I don't know.

Reply:
Using `pass` makes pylint unhappy and gives `unnecessary-pass`.

Originally I used `pass` at https://github.com/apache/superset/runs/2931421422 but I later removed by https://github.com/apache/superset/pull/15403/commits/822e2f3168670c95284cf606f3ac47ec372684b0 to make pylint happy.

Reply:
Huh. Maybe you don't need `pass` if there's a docstring

Reply:
Right, a docstring is a valid expression, so need for `pass`. It does look weird, though.

Reply:
@koszti shouldn't the base method return `False`?